common/batcher/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "batcher"
+version = "0.1.0"
+authors = ["Chris Greenaway <chris.greenaway@softwareag.com>"]
```

Comment:
May be need to change the authors to "Software AG <thin-edge-team@softwareag.com>

Reply:
I will remove it completely.